bin/json-refs

```diff
@@ -29,7 +29,7 @@
 // Set here so that commander does the right thing
 process.stdout.columns = 120;
 
-var JsonRefs = require('..');
+var JsonRefs = require('../dist/json-refs.cjs');
```

Comment:
The main reason for this PR was to be able to get it to build in order to start making further changes. I couldn't with the gulp infrastructure as it just crashes on my system, and gulp is no longer maintained, so I my next step to use something more modern.

I've used [preconstruct](https://preconstruct.tools/) to take care of the transpilation (including babel) for packaging. The switch to export/import is so that people using ES6 and bundlers now have a ESM compatible version of the code they can work with (the `.esm.js` extension). The `.cjs.js` extension refers to commonjs, which is the `module.exports` system that was in place before. Consumers of the code now have both.